templates/projects/task_presenter_editor.html

```diff
@@ -8,6 +8,11 @@
 <link rel="stylesheet" href="{{url_for('static', filename='vendor/codemirror/codemirror.css')}}"/>
 <link rel="stylesheet" href="{{url_for('static', filename='vendor/codemirror/addons/display/fullscreen.css')}}"/>
 <link rel="stylesheet" href="{{url_for('static', filename='vendor/codemirror/addons/dialog/dialog.css')}}"/>
+<link rel="stylesheet" href="{{url_for('static', filename='vendor/summernote/summernote.css')}}"/>
+<link  href="{{url_for('static', filename='vendor/summernote/font/summernote.eot')}}"/>
+<link  href="{{url_for('static', filename='vendor/summernote/font/summernote.ttf')}}"/>
+<link  href="{{url_for('static', filename='vendor/summernote/font/summernote.wolf')}}"/>
```

Comment:
Shouldn't the file extension be `.woff`?

Reply:
be careful with the wolf!